joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/ExternalCommand.scala

```diff
@@ -26,16 +27,24 @@ object ExternalCommand {
     }
   }
 
-  def run(
+  def runWithDir(
```

Comment:
This looks like the below function, just that `cwd` is optional. Wouldn't it make sense to make this implementation function private, then have a version of this that excludes the `cwd` parameter? I'm also aware that some of these defaults have to be excluded because they look the same to the compiler. Tricky...

Reply:
Was still playing around with this one, but will give that a try.